### PR TITLE
Fixes #8236 - Initialize bots from JSON data files

### LINKED ISSUE #8236: Initialize bots from JSON data files
**Feature**
#4199 

**Describe the task**
Use JSON data files to initialize the bots during start up

### PR DESCRIPTION
**Pull request by sureshms**
### Describe your changes :

See #8236

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by sureshms** [comment]
One more attempt. @pmbrull and @ShaileshParmar11 - please review python and UI changes

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Comment by ShaileshParmar11** [comment]
API is throwing an error while deploying can you please check?
cc @sureshms @pmbrull

**Comment by pmbrull** [comment]
Taking a look, thanks!

**Comment by sureshms** [comment]
> API is throwing an error while deploying can you please check? cc @sureshms @pmbrull

The updated patch should fix this

**Review by ShaileshParmar11 — APPROVED** [review]

**Comment by sureshms** [comment]
@pere committing the change. If you have any comments we can address that in a subsequent jira.

File: bootstrap/sql/com.mysql.cj.jdbc.Driver/v006__create_db_connection_info.sql [hunk 1]
```diff
@@ -9,4 +9,7 @@ CREATE TABLE IF NOT EXISTS web_analytic_event (
     deleted BOOLEAN GENERATED ALWAYS AS (json -> '$.deleted'),
     UNIQUE(name),
     INDEX name_index (name)
-);
\ No newline at end of file
+);
+
+UPDATE bot_entity
+SET json = JSON_INSERT(JSON_REMOVE(json, '$.botType'), '$.provider', 'system');
```
Comment (nahuelverdugo) [inline bootstrap/sql/com.mysql.cj.jdbc.Driver/v006__create_db_connection_info.sql:15]: This is not covering the case where the user has already created some bots. It assumes all the bots in the DB are the system's bots.

File: ingestion/src/metadata/ingestion/ometa/ometa_api.py [hunk 2]
```diff
@@ -180,7 +179,7 @@ def __init__(self, config: OpenMetadataConnection, raw_data: bool = False):
 
         # Load auth provider config from Secret Manager if necessary
         self.secrets_manager_client.add_auth_provider_security_config(
-            self.config, BotType.ingestion_bot.value
+            self.config, "ingestion-bot"
```
Comment (nahuelverdugo) [inline ingestion/src/metadata/ingestion/ometa/ometa_api.py:182]: do we want to hardcode the name everywhere?

**Pull request merged**